.github/workflows/bazelBuildAndTest.yml

```diff
@@ -66,7 +66,7 @@ jobs:
                    -v "$(pwd)":"/opt/src/torch-mlir" \
                    -v "${HOME}/.cache/bazel":"/root/.cache/bazel" \
                    torch-mlir:ci \
-                   bazel test @torch-mlir//test/...
+                   bazel test --test_output=errors @torch-mlir//...
```

Comment:
add `--test_output=errors ` for bazel test ci ,otherwise it can't not show the errors in console but some log file which we can't not access.